dot/config_integration_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//go:build integration
+// +build integration
+
+package dot
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestExportConfig tests exporting a toml configuration file
+func TestExportConfig(t *testing.T) {
+	cfg, cfgFile := NewTestConfigWithFile(t)
+	require.NotNil(t, cfg)
+
+	genFile := NewTestGenesisRawFile(t, cfg)
+	require.NotNil(t, genFile)
+
+	cfg.Init.Genesis = genFile.Name()
+
+	err := InitNode(cfg)
+	require.NoError(t, err)
+
+	file := ExportConfig(cfg, cfgFile.Name())
+	require.NotNil(t, file)
```

Comment:
Do you need to close that file perhaps?

Reply:
Yes, added os.Remove

Reply:
I meant `file.Close()` before removing it to clear the file handle. Not sure if removing it clears the file handle too.